src/lib/api/org/index.ts

```diff
@@ -275,3 +276,81 @@ function convertToSnykProject(projectData: v3ProjectData[]) : SnykProject[] {
   return projects
 }
 
+export async function listTargets(
+  requestManager: requestsManager,
+  orgId: string,
+): Promise<TargetsResponse> {
+  getApiToken();
+  getSnykHost();
+  debug(`Listing all targets for org: ${orgId}`);
+
+  if (!orgId) {
+    throw new Error(
+      `Missing required parameters. Please ensure you have set: orgId, settings.
+        \nFor more information see: https://apidocs.snyk.io/?version=2022-09-15%7Ebeta#get-/orgs/-org_id-/targets`,
+    );
+  }
+  
+  const targets = await listAllSnykTargetREST(requestManager, orgId)
+
+  return {targets};
+}
+
+export async function listAllSnykTargetREST(requestManager: requestsManager,
+  orgId: string,
+  ): Promise<SnykTargetREST[]> {
+
+    let lastPage = false;
+    let snykTargetRESTList: SnykTargetREST[] = [];
+    let pageCount = 1;
+    let nextPageLink: string | undefined = undefined;
+    while (!lastPage) {
+      try {
+        const {
+          SnykTargetREST,
+          next,
+        }: {
+          SnykTargetREST: SnykTargetREST[];
+          next?: string;
+        } = await getSnykTargetREST(requestManager, orgId, nextPageLink);
+
+        snykTargetRESTList = snykTargetRESTList.concat(SnykTargetREST);
+        next
+          ? ((lastPage = false), (nextPageLink = next))
+          : ((lastPage = true), (nextPageLink = ''));
+        pageCount++;
+      } catch (e) {
+        debug('Failed to get targets for ', orgId, e);
+        throw e;
+    }
+  }
+  return snykTargetRESTList
+}
+
+export async function getSnykTargetREST(requestManager: requestsManager,
```

Comment:
please see comment here to implement filtering targets https://github.com/snyk-tech-services/snyk-api-import/pull/360/files#:~:text=lili2311-,now,-missing%20all%20query